appengine/standard/firebase/firetactoe/fire_index.html

```diff
@@ -0,0 +1,304 @@
+<html>
+  <head>
+
+  <script src="https://www.gstatic.com/firebasejs/3.3.2/firebase.js"></script>
+  <!-- <script src="https://www.gstatic.com/firebasejs/3.3.0/firebase-database.js"></script> -->
+  <script>
+  	// Initialize Firebase object
+  	var config = {
+    	apiKey: "AIzaSyB_pwNiBi8Q3iakJitkQHtEQ5SZWmp9EiY",
+    	authDomain: "imposing-mind-131903.firebaseapp.com",
+    	databaseURL: "https://imposing-mind-131903.firebaseio.com"
```

Comment:
probably best to put placeholders here, preferably put all this stuff in a single file that gets pulled in somehow.